FlowCrypt/Functionality/Mail Provider/Imap/Imap+session.swift

```diff
@@ -76,9 +78,14 @@ extension Imap {
 
     func disconnect() {
         let start = DispatchTime.now()
```

Comment:
Am I right that it was for purposes of measuring time between for example disconnect started and finished?

Reply:
@tomholub

Reply:
Correct, start of action to end of action

Reply:
Let me came up with better solution for his purpose? CFAbsoluteTime is better for this purpose

Reply:
Sure. The point is just to figure out latency of each action, it's up to you how you think it's best achieved.